### ISSUE TITLE
Problem testing components using Radium

### ISSUE THREAD

**Issue by eriklharper** [FormidableLabs/radium#417]
I'm writing a component in ES5 (compiled with react-tools) that uses Radium and it works fine in the browser.  However my test for the component is throwing a SyntaxError when I run my tests (using Jest):

```
SyntaxError: /x/igniter5/widget-library/src/components/youtube-carousel/tests/youtube-carousel-test.js: /x/igniter5/widget-library/src/components/youtube-carousel/youtube-carousel.js: /x/igniter5/widget-library/src/components/youtube-carousel-thumbnail/youtube-carousel-thumbnail.js: /x/igniter5/widget-library/node_modules/radium/lib/index.js: /x/igniter5/widget-library/node_modules/radium/lib/enhancer.js: /x/igniter5/widget-library/node_modules/radium/lib/resolve-styles.js: /x/igniter5/widget-library/node_modules/radium/src/__mocks__/exenv.js: Unexpected reserved word
```

This is strange as it seems like its balking on possibly ES6 language features, which doesn't compute since I'm using `babel-jest` which should not have an issue with ES6.

**Comment by ianobermiller** [comment]
Ah, I wonder if Jest is overreaching a bit here and trying to load things it shouldn't, due to the automocking. I should make sure to npmignore the tests and mocks anyway. But you might want to tweak your setup to ignore such files if you can.

**Comment by eriklharper** [comment]
Here you go @ianobermiller : https://github.com/eriklharper/radium-jest-repro.  Pull that down, run `npm install` and then `jest` and you should see the SyntaxError I mentioned above.  Also, keep in mind I'm running this locally with `Node v4.1.2`.

**Comment by eriklharper** [comment]
I'm using Radium's features on this line for the component in question: https://github.com/eriklharper/radium-jest-repro/blob/master/src/components/youtube-carousel-thumbnail/youtube-carousel-thumbnail.js#L88

**Comment by ianobermiller** [comment]
@eriklharper, could you be running into the same issues as #420?

**Comment by ianobermiller** [comment]
@eriklharper Did you manage to solve this? I tried to repro a few different ways with your repo, and keep getting different errors, but never a syntax error :S

**Comment by eriklharper** [comment]
@ianobermiller thanks for the follow-up.  I was able to see the actual test errors after nombom'ing the project, so this has unblocked the original issue I had on this thread! Whew, I wasn't going crazy.

After I nombom'ed and everything was reinstalled, I noticed that the problem file (`/x/igniter5/widget-library/node_modules/radium/src/__mocks__/exenv.js`) was not downloaded, which confirms what you said above about ignoring test mock files.

Thanks!
